internal/eventstore/handler/v2/handler.go

```diff
@@ -111,11 +112,19 @@ func (h *Handler) Start(ctx context.Context) {
 }
 
 func (h *Handler) schedule(ctx context.Context) {
-	// if there was no run before trigger instantly
-	t := time.NewTimer(0)
+	// if there was no run before trigger within half a second
+	start := randomizeStart(0, 0.5)
+	t := time.NewTimer(start)
 	didInitialize := h.didProjectionInitialize(ctx)
 	if didInitialize {
-		t.Reset(h.requeueEvery)
+		if !t.Stop() {
+			<-t.C
+		}
+		// if there was a trigger before, start the projection
+		// within a second (should generally be after the not initialized projections)
```

Comment:
After a second, right? Min == 1 sec, Max == requeueEvery setting.

```suggestion
		// after a second (should generally be after the not initialized projections)
```

Reply:
it's within a second and `RequeueEvery `, but yes `after` is easier to read / understand